cityofaustin/janis
# Issue 4327: Change hard-coded content in Trashy App

https://github.com/cityofaustin/techstack/issues/4327

# Description

Fixes the hardcoded message per Amanda's instructions.

# Types of changes



# How can this be tested?
https://janis-v3-4327-trashy.netlify.app/housing-utilities/recycling-trash-and-compost/household-waste/bulk-item-pickup/

Enter an address into step 3. You should see the "Currently, no upcoming bulk item..." message above the list of dates.

# Checklist:
